src/ex4/server/middlewares/logger.js

```diff
@@ -0,0 +1,6 @@
+async function logger(req, res, next) {
```

Comment:
would rename the middleware to a "requestLogger" as "logger" is too generic name. the purpose of the middleware is to log data regarding an incoming request.